cucumber/authenticators_jwt/features/step_definitions/authn_jwt_steps.rb

```diff
@@ -0,0 +1,13 @@
+Given(/I successfully set auth-jwt jwks-uri variable with value of "([^"]*)" endpoint/) do |filename|
+  create_jwt_secret("jwks-uri", "#{JWTJWKsHelper::JWKS_BASE_URI}/#{filename}")
+end
+
+Given(/I successfully set auth-jwt token-app-property variable value "([^"]*)"/) do |value|
+  create_jwt_secret("token-app-property", value)
+end
```

Comment:
these 2 steps can be merged into one where you get the variable name as a property. feel free to leave it like this if you think it is better.

Reply:
I prefer to keep is separate. It provides more flexibility and pure building blocks also for `provider-uri` use case